test/integration/nav_test.rb

```diff
@@ -3,6 +3,9 @@
 class NavTest < ActionDispatch::IntegrationTest
   def setup
     auth_setup
+    @registrar = registrar(:valid)
```

Comment:
If this is only needed for the registrar tests, you might want to either have a separate method that does that that is called from the tests that need it, or just put this setup code into the tests that need it.

Setting up registrar stuff for non-registrar tests feels a bit confusing to me.